src/Relation/Nullary.agda

```diff
@@ -25,18 +26,22 @@ infix 2 _because_
 ¬_ : ∀ {ℓ} → Set ℓ → Set ℓ
 ¬ P = P → ⊥
 
+------------------------------------------------------------------------
 -- `Reflects` idiom.
+
 -- The truth value of P is reflected by a boolean value.
 
 data Reflects {p} (P : Set p) : Bool → Set p where
   ofʸ : ( p :   P) → Reflects P true
   ofⁿ : (¬p : ¬ P) → Reflects P false
 
--- Decidable relations.
--- This version of `Dec` allows the boolean portion of the
--- value to compute independently from the proof portion.
--- This often allows good computational properties when we
--- only care about the boolean portion.
+------------------------------------------------------------------------
+-- Decidability.
+
+-- This definition allows the boolean portion of the value to compute
+-- independently from the proof portion. This translates to better
+-- computational properties when we only care about the result and not
+-- the proof. See README.Decidability for further details.
```

Comment:
Looking back, I think “the value” should be something more informative, like “a decision procedure”.

Reply:
Updated the text.